specification/analysisservices/resource-manager/Microsoft.AnalysisServices/stable/2017-08-01/analysisservices.json

```diff
@@ -1526,50 +1519,13 @@
       },
       "description": "The resource management error additional info."
     },
-    "ErrorObject": {
-      "type": "object",
-      "description": "The error object",
-      "properties": {
-        "code": {
-          "description": "Error code",
-          "type": "string"
-        },
-        "message": {
-          "description": "Error message indicating why the operation failed.",
-          "type": "string"
-        },
-        "subCode": {
-          "description": "The error sub code",
-          "type": "integer",
-          "format": "int32"
-        },
-        "httpStatusCode": {
-          "description": "The http status code",
-          "type": "integer",
-          "format": "int32"
-        },
-        "timeStamp": {
-          "description": "the timestamp for the error.",
-          "type": "string",
-          "readOnly": true
-        },
-        "details": {
-          "readOnly": true,
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/ErrorDetail"
-          },
-          "description": "The error details."
-        }
-      }
-    },
     "ErrorResponse": {
       "description": "Describes the format of Error response.",
       "type": "object",
       "properties": {
         "error": {
           "description": "The error object",
-          "$ref": "#/definitions/ErrorObject"
+          "$ref": "#/definitions/ErrorDetail"
```

Comment:
Recommend referring to common definitions "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/..." This will enforce the right schema.